lib/mayaUsd/render/mayaToHydra/utils.cpp

```diff
@@ -13,6 +13,18 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 //
+
+// GLEW must be included early (for core USD < 21.02), but we need pxr.h first
+// so that USD_VERSION_NUM has the correct value.
+// We also disable clang-format for this block, since otherwise v10.0.0 fails
+// to recognize that "utils.h" is the related header.
+// clang-format off
```

Comment:
Another `clang-format` oddity here, but when I added this block of includes and the conditional, `clang-format` v10.0.0 starts failing to recognize that `"utils.h"` is the related header for this cpp file, and it tries to put it at the bottom of the "private" header group at line 32. I opted to disable `clang-format` here rather than absorb that change.

Reply:
Do you happen to know if v11.0.0 will has it fixed?

Reply:
> Do you happen to know if v11.0.0 will has it fixed?

I just built clang v11.0.0 and gave it a try (using the current configuration) and I get the same results. It seems to get tripped up by the conditional that wraps the `glew.h` `#include`. If I remove the `#if`/`#else`, it correctly figures out that `utils.h` should stay where it is, just after `glew.h` but before the other "local" headers.